src/main/java/org/yomigae/model/SpotlightModel.java

```diff
@@ -0,0 +1,65 @@
+package org.yomigae.model;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.common.collect.ImmutableList;
+
+import heronarts.lx.model.LXModel;
+import heronarts.lx.model.LXPoint;
+import heronarts.lx.transform.LXTransform;
+
+public class SpotlightModel extends LXModel {
```

Comment:
This is totally pedantic, but since I've been reading a lot about this lately: technically what we're using are PAR washers--a spotlight is actually a different kind of light.

I have no strong compulsion to change the class name based on this. :P